hnn_core/cell.py

```diff
@@ -548,6 +550,39 @@ def record_soma(self, record_vsoma=False, record_isoma=False):
         if record_vsoma:
             self.rec_v.record(self._nrn_sections['soma'](0.5)._ref_v)
 
+    def record_sec(self, record_vsec=False, record_isec=False):
```

Comment:
soma is also a section in the Neuron scheme of things. Would it make sense to unify the API with the recording of soma?

Reply:
For sure! While I'm still testing things I want to keep the code separate until I know recordings work for all sections. Then I can delete all the soma specific code (and set an option between recording just the soma, or all sections).